packages/dev/core/src/Rendering/depthRenderer.ts

```diff
@@ -192,7 +192,7 @@ export class DepthRenderer {
 
             // Culling
             const detNeg = effectiveMesh._getWorldMatrixDeterminant() < 0;
-            let sideOrientation = renderingMesh.overrideMaterialSideOrientation ?? material.sideOrientation;
+            let sideOrientation = material.sideOrientation ?? renderingMesh.sideOrientation;

```

Comment:
We have a problem if both `material.sideOrientation` and `renderingMesh.sideOrientation` are `null`... We would have to get a default value, based on `scene.useRightHandedSystem`, which is a bit convoluted...

Let's discuss it with @sebavan and @bghgary, we should probably rethink a bit how we want to deal with this side orientation problem.